addOns/retest/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ## Unreleased
 ### Changed
 - Update minimum ZAP version to 2.11.1.
+- Plugin Id values are no longer displayed with commas.
```

Comment:
`Scan Rule ID`, I know it's not what's currently shown but we should correct that (if not in this PR in other).

Reply:
I could tweak that here too.

Reply:
This one was not changed, on purpose?

Reply:
Oh no, I just looked past it somehow. I'll do this one too but it won't happen till the morning for me.